src/main/java/com/mitchej123/hodgepodge/client/textures/Mipmaps.java

```diff
@@ -0,0 +1,25 @@
+package com.mitchej123.hodgepodge.client.textures;
+
+public class Mipmaps {
+
+    private static final float[] VALS = new float[256];
```

Comment:
Please, give this variable more meaningful name (`CACHED_CHANNEL_VALUES`, for example).